### ISSUE TITLE
rubocop ignores .ruby-version when run from a subdirectory

### ISSUE THREAD

**Issue by aclemons** [bbatsov/rubocop#4601]
I use flycheck from emacs to run rubocop which invokes it from the directory of the file it is linting. Additionally, I have a `.ruby-version` in my project root, and no `TargetRubyVersion` set in my rubocop config.

In this case, rubocop ignores my `.ruby-version` when invoked from the subdirectory and uses the default ruby version.

Example:
```
$ rubocop -V
0.48.1 (using Parser 2.4.0.0, running on ruby 2.3.4 x86_64-linux)
$ cat .ruby-version 
2.3.4
$ cat lib/thing.rb 
# frozen_string_literal: true

class Thing
  def initialize(foo)
    @foo = foo
  end

  def bar
    foo&.bar
  end
end
$ rubocop lib/thing.rb 
Inspecting 1 file
.

1 file inspected, no offenses detected
$ cd lib
$ rubocop thing.rb 
Inspecting 1 file
E

Offenses:

thing.rb:9:8: E: unexpected token error
(Using Ruby 2.1 parser; configure using TargetRubyVersion parameter, under AllCops)
    foo&.bar
       ^^

1 file inspected, 1 offense detected
```

**Comment by rrosenblum** [comment]
I think this is a duplicate of #4186.

**Comment by aclemons** [comment]
Indeed. I searched for an existing issue, but apparently not well enough. Thanks.
